packages/backend/server/src/plugins/copilot/controller.ts

```diff
@@ -279,7 +231,7 @@ export class CopilotController implements BeforeApplicationShutdown {
       const finalMessage = session.finish(params);
       info.finalMessage = finalMessage.filter(m => m.role !== 'system');
 
-      const content = await provider.generateText(finalMessage, model, {
+      const content = await provider.text({ modelId: model }, finalMessage, {
```

Comment:
If you have already selected a provider that matches the input and output in `chooseProvider`, is it unnecessary to check again in text?

Reply:
communicated